code/modules/clothing/clothing.dm

```diff
@@ -217,6 +217,23 @@
 		for(var/A in armor)
 			armor[A] -= rand(armor[A]/3, armor[A])
 
+/obj/item/clothing/attack(mob/living/M, mob/living/user, def_zone, var/originator = null)
```

Comment:
Another nitpick, I live all vars in arguments to be prefixed with `var/`. That would look like `var/mob/living/M`. Again, purely a style change, feel free not to do this.

Reply:
Stop

Reply:
>Collaborator arguing with a member
What does Space Law says about this

Reply:
I think it was decided a while back that you can't decide something is a standard without first actually standardizing it
I think a regex to change all parameters to one format or another would be pretty easy, but until today I wasn't aware even a single person cared

Reply:
Yeah, absolutely, I wanted to refrain from posting this but ehh